espnet2/enh/layers/complexnn.py

```diff
@@ -0,0 +1,436 @@
+import torch
```

Comment:
I missed this function.
This is really cool!
Please add the description about dccrn and this new function in https://github.com/espnet/espnet#se-speech-enhancement-and-separation

Reply:
I have added the DCCRN and the link of the original paper in the README